runelite-client/src/main/java/net/runelite/client/plugins/worldmap/QuestStartLocation.java

```diff
@@ -27,151 +27,162 @@
 
 import lombok.Getter;
 import net.runelite.api.coords.WorldPoint;
+import net.runelite.api.Quest;
 
+// TODO: add locations to net.runelite.api.Quest, and collect
```

Comment:
I don't see any reason for `Quest` to contain the start point, would just remove this todo

Reply:
It collapses these two enums (which are just joined via the quest id) into one.  also makes it so you can't add a new Quest and forget to add a location.

Reply:
`Quest` is just supposed to allow plugins to uniquely identify quests. The world map plugin cares about the start point so it can add an enum for each quest to contain the extra data it needs - but I don't see why that should be in the core api.

Reply:
I'm actually surprised that other plugins haven't found a reason to use quest locations. Something related to route planning or something. Since nothing else uses it, I appreciate the argument that it shouldn't be in core.